src/hotspot/share/gc/z/zPageCache.cpp

```diff
@@ -278,18 +280,20 @@ class ZPageCacheFlushForUncommitClosure : public ZPageCacheFlushClosure {
 private:
   const uint64_t _now;
   uint64_t*      _timeout;
+  uintx          _delay;
 
 public:
-  ZPageCacheFlushForUncommitClosure(size_t requested, uint64_t now, uint64_t* timeout)
+  ZPageCacheFlushForUncommitClosure(size_t requested, uint64_t now, uint64_t* timeout, uintx delay)
     : ZPageCacheFlushClosure(requested),
       _now(now),
-      _timeout(timeout) {
+      _timeout(timeout),
+      _delay(delay) {
     // Set initial timeout
     *_timeout = ZUncommitDelay;
```

Comment:
Shouldn't this also use `delay`? This shouldn't have any real influence now (the code that has `delay != ZUncommitDelay` doesn't use the timeout) but anyway.